nf_core/launch.py

```diff
@@ -427,28 +438,48 @@ def prompt_group(self, group_id, group_obj):
         """
         while_break = False
         answers = {}
+        error_msgs = []
         while not while_break:
+
+            if len(error_msgs) == 0:
+                self.print_param_header(group_id, group_obj, True)
+
             question = {
                 "type": "list",
                 "name": group_id,
-                "message": group_obj.get("title", group_id),
+                "qmark": "",
+                "message": "",
+                "instruction": " ",
                 "choices": ["Continue >>", questionary.Separator()],
             }
 
+            # Show error messages if we have any
+            for msg in error_msgs:
+                question["choices"].append(
+                    questionary.Choice(
+                        [("bg:ansiblack fg:ansired bold", " error "), ("fg:ansired", f" - {msg}")], disabled=True
+                    )
+                )
+            error_msgs = []
```

Comment:
Wouldn't that make more sense after line 491, where you check for errors?
Or does it require a 'fresh' question object?

Reply:
No, the logic is a bit funny here. If there are errors then the question is asked again. So in lines 491 we collect error messages, then the loop fires again. Here we are printing the errors if there are any from the last time we went around and then clearing them once shown (so that they don't persist forever).

Reply:
Ah I get it, makes sense like that :D thanks!